### DISCUSSION TITLE
laughedelic/coursier-setup -> coursier/setup-action?

### DISCUSSION THREAD

**Discussion by laughedelic** [coursier/coursier#2056] [Ideas]
Hi! I didn't know what is the best place to post this, hope it's ok here.

@alexarchambault what do you think about transferring https://github.com/laughedelic/coursier-setup to the Coursier GitHub org? And maybe renaming it to `coursier/setup-action` to be in line with `coursier/cache-action`?

I was hoping that Coursier will be included in the list of preinstalled tools on GitHub Actions runners (and I will deprecate the action), but in the light of the last response: https://github.com/actions/virtual-environments/issues/3121#issuecomment-817690558 it seems that the action is here to stay. So I thought maybe it should become official?

I've been using it myself in a few projects at work and I think it's fairly stable. There are a [few things](https://github.com/laughedelic/coursier-setup/projects/1) that could be improved, but nothing essential in my opinion.

I'm willing to keep maintaining it, but just thought it would be nice to transfer the ownership of the repo.

**Comment by alexarchambault** [comment]
Hi @laughedelic, I'm fine transferring the repository to https://github.com/coursier/setup-action. I just invited you in the coursier organization, which should allow you to move the repository there.

I've been using the extension for quite some time too, it's been working really fine 👍 

I'd love to make it more prominent on the coursier website too. Just so you know, I'm working on some overhaul of the build of coursier right now. As soon as it's done, a new release and some rework of the website should be next, and I'll add a page about the GitHub actions then.

**Reply by laughedelic** [reply]
Sounds great! 

Thanks for adding me to the org. I transferred the repo: https://github.com/coursier/coursier-setup, but after that lost access to its settings, so couldn't rename. Could you add me as the repo admin or maintainer?

**Reply by alexarchambault** [reply]
Just gave you admin rights on it.

**Reply by laughedelic** [reply]
Strange, but the settings page looks different now, and I can't rename it.

<details><summary>screenshot</summary>

<img width="1104" alt="Screenshot 2021-06-17 at 17 07 49" src="https://user-images.githubusercontent.com/766656/122423878-8d6d7a80-cf8e-11eb-90d0-1ccf8f3426d9.png">

</details>

**Reply by alexarchambault** [reply]
Just changed stuff in the organization member privileges settings, which may have fixed this.

**Reply by laughedelic** [reply]
Still the same, unfortunately. I think it's fine as is, if you can just rename it.

**Reply by alexarchambault** [reply]
Ok, just did the rename.
